e2e/testcases/gatekeeper_test.go

```diff
@@ -30,6 +30,10 @@ func TestConstraintTemplateAndConstraintInSameCommit(t *testing.T) {
 	// TODO enable the test on autopilot clusters when GKE 1.21.3-gke.900 reaches regular/stable.
```

Comment:
Out of scope for this change, but looks like we can address this TODO

Reply:
Good catch. Let us fix this in a separate PR.